benchmarks/block_ref.py

```diff
@@ -0,0 +1,90 @@
+import click
+import aiosqlite
+import asyncio
+import time
+import random
+
+from typing import Optional, List
+from pathlib import Path
+from dataclasses import dataclass
+
+from chia.consensus.blockchain import Blockchain
+from chia.consensus.default_constants import DEFAULT_CONSTANTS
+from chia.full_node.block_store import BlockStore
+from chia.full_node.coin_store import CoinStore
+from chia.full_node.hint_store import HintStore
+from chia.types.blockchain_format.program import SerializedProgram
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.util.db_version import lookup_db_version
+from chia.util.db_wrapper import DBWrapper
+from chia.util.ints import uint32
+from valid_block_heights import transaction_block_heights
```

Comment:
this list is quite long, and hard coded to our actual chain up until recently. Is there a better way to do this other than including a very large python file?
I don't want to compute it in the benchmark for 2 reasons. 1. It would take a long time. 2. it would bring in blocks into the cache and make the test itself less accurate. (the difference between cold and hot cache on HDD is enormous, 12 seconds -> 3.5 seconds).

Reply:
Hi